### DISCUSSION TITLE
Function `wpc2o_update_order_notes` not found or invalid function

### DISCUSSION THREAD

**Discussion by c3aus3scu** [AshleyRedman/WPClothes2Order#89] [Q&A] [answered]
Hi. After I add an product and order, after in the order info under the Customer tab, I received this error: Warning: call_user_func_array() expects parameter 1 to be a valid callback, function 'wpc2o_update_order_notes' not found or invalid function name in /home/daemons/public_html/wp-includes/class-wp-hook.php on line 308

**Comment by AshleyRedman** [comment]
@c3aus3scu Thanks for reporting, are you able to let me know the following:

- What version of php are you running
- Where abouts are you seeing this error, i.e thank you page or after clicking the order button etc
- Are you able to provide some screenshots or recordings of this flow?

**Reply by AshleyRedman** [reply]
After testing, I was able to recreate, a PR will be update today as a hotfix. Will comment when thats done.

**Comment by AshleyRedman** [comment, marked as answer]
@c3aus3scu Hot fix has been deployed. Please update your plugin from `1.0.3` to the latest `1.0.4` release.

**Comment by c3aus3scu** [comment]
Thank you very quick replay, now is ok no more errors now after a placed an order I receive this: <html>

634 Marius-Vasile Adam
1 min ago	
Order failed
Processing

**Reply by AshleyRedman** [reply]
Can you confirm that your api key is correctly entered & the post order endpoint has been provided? If either of those two values are incorrect, the order request to C2O would fail

**Reply by c3aus3scu** [reply]
Yes I use: https://www.clothes2order.com/api/post-order/ for the endpoint
And the API is copy pasted.
Maybe you can have a look? I will let you have an administrator account.

**Reply by AshleyRedman** [reply]
@c3aus3scu If you could send me an email, I can take a look to see where the issue is coming from

**Comment by c3aus3scu** [comment]
@AshleyRedman you have an email on ash.redman@outlook.com

**Reply by AshleyRedman** [reply]
Do you mind resending?

**Reply by AshleyRedman** [reply]
@c3aus3scu Please open a new discussion as this one is marked as resolved as the original issue is resolved and it sounds like you have having a different issue.
